perfkitbenchmarker/benchmarks/cloudsuite_web_search_benchmark.py

```diff
@@ -59,27 +59,27 @@
              '$FABAN_HOME/lib/fabandriver.jar:$JAVA_HOME/lib/tools.jar:'
              '$FABAN_HOME/search/build/lib/search.jar')
 
-flags.DEFINE_string('client_heap_size', '2g',
+flags.DEFINE_string('websearch_client_heap_size', '2g',
```

Comment:
We still have some naming inconsistency here...

websearch vs cloudsuite_web_search

I've ok if we want to shorten this but we should be consistent on websearch or web_search and either rename the flag or benchmark.

My preference is websearch